pkg/controller/persistentvolume/persistentvolume_claim_binder_controller.go

```diff
@@ -166,6 +192,11 @@ func syncVolume(volumeIndex *persistentVolumeOrderedIndex, binderClient binderCl
 		volumeIndex.Add(volume)
 	}
 
+	if isBeingProvisioned(volume) {
```

Comment:
Let's say the provisioner controller picked up the PVC first and then this binder controller logic is triggered. The provisioner controller will set the `pvProvisioningRequiredAnnotationKey` annotation to `pvProvisioningCompletedAnnotationValue` but the claim is not actually bound yet since the volume isn't actually provisioned. But since this code is also executing `isBeingProvisioned(...)`, will return false. Should this code continue in that case?